lesson03/main.js

```diff
@@ -0,0 +1,17 @@
+const ul = document.getElementById("js-target");
+const imgSrcs = ["/img/bookmark.png","/img/messege.png"];
+
+for (let i=0; i < imgSrcs.length; i++){
+  const li = document.createElement("li");
+  const a = document.createElement("a");
+  const img = document.createElement("img");
+
+  a.href = `a${i+1}.html`;
+  a.textContent = `a${i+1}`;
+  img.src = imgSrcs[i];
+
+  ul.appendChild(li).appendChild(a).insertAdjacentElement("beforebegin",img);
```

Comment:
MUST
The following modifications are required because the DOM structure is different from the specification.
```suggestion
  ul.appendChild(li).appendChild(a).insertAdjacentElement("afterbegin",img);
```

Reply:
Thank you mai-san:)
I learned insertAdjacentElement!
c0017687fd2f89161209216657a74fa82f1b36a5